content/en/blog/2024-05-16-modelix-platform-release-24.1/index.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2024-05-16
+title: "Modelix Platform Release 24.1"
+---
+
+Version **`24.1`** of the modelix platform is out now!
+
+The versions bundled in this release are:
+
+```
+modelixCoreVersion = 7.4.0
+incrementalVersion = 0.2.0
+buildToolsVersion = 1.5.0
+mpsPluginsVersion = 0.7.0
```

Comment:
Wouldn't it be easier for the users to have actual artifact names (and links to some repo) here instead of these abstract versions?

Reply:
`mpsPluginsVersion` shall be updated if https://github.com/modelix/modelix.mps-plugins/pull/100 is merged before release. (Which I suggest to happen.)

Reply:
@languitar it's the way we've always done it, we can discuss this for future releases.

Reply:
Why not improving this right now?

Reply:
I added a link that will point to the versions toml, once it is released. 
If someone really needs the information that detailed then that should suffice.